static/styles/compose.scss

```diff
@@ -377,16 +377,16 @@ input.recipient_box {
     margin-right: 2px;
 }
 
-#send_controls #compose-send-button {
+#compose-send-button {
     border-radius: 4px;
-    padding-top: 3px;
-    padding-bottom: 3px;
     font-weight: 600;
     border: 1px solid hsl(170, 68%, 41%);
     background-color: hsl(170, 48%, 54%);
     color: hsl(0, 0%, 100%);
+    padding: 3px 10px;
```

Comment:
I prefer to more explicitly set the four padding borders rather than the shorthand syntax.